src/promptflow-evals/promptflow/evals/synthetic/_conversation/__init__.py

```diff
@@ -85,9 +85,12 @@ def __init__(
                 if isinstance(conversation_starter_content, dict):
                     self.conversation_starter = conversation_starter_content
                 else:
-                    self.conversation_starter = jinja2.Template(
-                        conversation_starter_content, undefined=jinja2.StrictUndefined
-                    )
+                    try:
+                        self.conversation_starter = jinja2.Template(
+                            conversation_starter_content, undefined=jinja2.StrictUndefined
+                        )
+                    except jinja2.exceptions.TemplateSyntaxError as e:  # noqa: F841

```

Comment:
If this is due to user error then do we just use the template with error ?

Reply:
`conversation_starter_content` is coming from the RAI service

Reply:
Is it expected that template can be syntactically incorrect ?